lib/whitehall/document_filter/options.rb

```diff
@@ -76,6 +77,13 @@ def options_for_topics
         @options_for_topics ||= StructuredOptions.new(all_label: "All policy areas", grouped: Classification.grouped_by_type)
       end
 
+      def options_for_people
+        @options_for_people ||= StructuredOptions.new(
+          all_label: "All people",
+          ungrouped: Person.all.sort_by(&:name).map { |o| [o.name, o.slug] }
```

Comment:
I see why the `.order(:name).pluck(:name, :slug)` thing doesn't work now. `name` is a method not a db field 🤔 `.order` would probably still be faster before doing the map but I'm not sure it would be that significant. In digging about in this though I realise now that there are > 2800 people in the Whitehall database. That's a lot of options in a dropdown. The departments one has > 1000 too. This isn't good UI/UX. Hopefully this will get moved to finder-frontend in the next quarter.

Reply:
Here's hoping. It doesn't help that some people are ordered by title and not first name.